collections/_article/godotcon-2025-xr-android-recap.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "What's New in Android & XR"
+excerpt: "Recap of the Android & XR updates during GodotCon 2025"
+categories: ["events", "progress-report"]
+author: "Fredia Huya-Kouadio"
+image: /storage/blog/covers/godotcon-boston-2025-android-xr.webp
+date: 2025-05-30 18:00:00
+---
+
+# What’s New in XR & Android
+
+## GodotCon 2025 Recap
+
+A few weeks ago during [GodotCon Boston 2025](https://conference.godotengine.org/2025/), [David Snopek](https://github.com/dsnopek), [Logan Lang](https://github.com/devloglogan) and I gave the latest updates on the state of Android and XR for the Godot engine.  
+You can see the full presentation slides @ [https://bit.ly/godotcon-boston-2025-xr-android](https://bit.ly/godotcon-boston-2025-xr-android) 
+
+# Android & XR editor
+
+## Android Editor
+
+<img alt="Android editor on phone" src="/storage/blog/covers/godotcon-boston-2025-android-xr.webp" />
+
+An update on the Android editor is long overdue\! Since [its release in 2023](https://godotengine.org/article/android_godot_editor_play_store_beta_release/), the Android editor has been available on the [Google Play store](https://play.google.com/store/apps/details?id=org.godotengine.editor.v4) and on [Godot’s download page](https://godotengine.org/download/android/) with support for all Android devices (e.g: phones, foldables, tablets, chromebooks, others..). 
```

Comment:
```suggestion
An update on the Android editor is long overdue\! Since [its release in 2023](https://godotengine.org/article/android_godot_editor_play_store_beta_release/), the Android editor has been available on the [Google Play store](https://play.google.com/store/apps/details?id=org.godotengine.editor.v4) and on [Godot’s download page](https://godotengine.org/download/android/) with support for all Android devices (e.g.: phones, foldables, tablets, Chromebooks, and others). 
```